vector/build.gradle

```diff
@@ -86,8 +86,9 @@ android {
         }
     }
 
-    // Specifies a flavor dimension (no need to specify it in each productsFlavor, cause we only declare one dimension).
-    flavorDimensions "flavor_dimension_vector"
+    // The base dimension permits to deal with GooglePlay/Fdroid app
+    // The voip flavor dimension permits to include/exclude jitsi at compilation time
+    flavorDimensions "voip", "base"
```

Comment:
The order is not correct, the flavor dimension must be: "base", "voip"

Reply:
As depicted [here](http://google.github.io/android-gradle-dsl/current/com.android.build.gradle.AppExtension.html#com.android.build.gradle.AppExtension:flavorDimensions(java.lang.String[])) and [here](https://developer.android.com/studio/build/build-variants#flavor-dimensions), flavors are ordered by decreasing priority order (highest priority first); to have the voip flavor applied "after"/"above" the base flavor, it thus has to appear before. I agree it is not very intuitive...

Reply:
I'm sorry but `base` must be taken first.
Else the sources defined in /app (like google-service.json) are not considered

Currently the build withVoipAppDebug failed
Please reverse the order to consider base first

Reply:
Done